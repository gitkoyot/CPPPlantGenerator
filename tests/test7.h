#ifndef TEST7_H
#define TEST7_H

#include <string>
#include <vector>


class MainClassWithDependencyToStd
{
public:
	std::string publicString;
  	std::vector<std::string> publicVectorOfStrings;
};


#endif
