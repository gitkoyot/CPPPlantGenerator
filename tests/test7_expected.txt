@startuml
class MainClassWithDependencyToStd{
publicString   :   std::string
publicVectorOfStrings   :   std::vector<std::string>
}

MainClassWithDependencyToStd *-- std.vector
MainClassWithDependencyToStd *-- std.string
@enduml

